src/main/resources/application-prod.yml

```diff
@@ -0,0 +1,13 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://localhost:5432/b
+    driver-class-name: org.postgresql.Driver
+    username: postgres
+    password: 8557
```

Comment:
this is dangerous when we have public repozytorium in github. if you havnt idea how resolve this problem, i propose create file `application-prod.yml.dist`. next add info to readme:

**edit password to database**
```
cp src/main/resources/application-prod.yml.dist src/main/resources/application-prod.yml
nano src/main/resources/application-prod.yml
```

Reply:
file `src/main/resources/application-prod.yml.dist` is example. it is template file for manually edit

Reply:
one more important thing. you should add to .gitignore file: `src/main/resources/application-prod.yml` when you check this solution^